.rubocop.yml

```diff
@@ -128,8 +128,9 @@ RSpec/AnyInstance:
 
 RSpec/BeforeAfterAll:
   Exclude:
-    - 'spec/load_fixtures_helper.rb' # being careful to avoid state leakage
-    - 'spec/models/preservation_policy_spec.rb' # the after(:all) here specifically prevents state leakage
+    # we use after(:all) specifically to prevent state leakage
```

Comment:
haha - this is fine;  I was thinking of keeping the comments after the file name and then repeating it.